lib/kracken/controllers/token_authenticatable.rb

```diff
@@ -14,7 +14,7 @@ def self.included(base)
       end
 
       # NOTE: Monkey-patch until this is merged into the gem
-      def request_http_token_authentication(realm = 'Application')
```

Comment:
I am worried about that -- it might not, but the only option I can see is to conditionally define this method with different arity based on the Rails version.